examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -14853,6 +14693,38 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
+              "name": "UnoccupiedCoolingSetpoint",
+              "code": 19,
+              "mfgCode": null,
+              "side": "server",
+              "type": "int16s",
+              "included": 1,
```

Comment:
I'm surprised we're not seeing .matter file changes for this.  Should we be?

Reply:
zap file is removed